### DISCUSSION TITLE
navbar - toggle icons, when the according page is chosen

### DISCUSSION THREAD

**Discussion by DorisW3** [neuefische/questions#722] [Web] [answered]
Hey  everyone,

I currently do some styling in my project and struggle with adding a feature to the navbar. When a navbar icon is clicked there should be another icon, as long as the user is on the according page. The present code doesn't work (at first, I tried it only with the favorite page), but I don't know why.

I searched in the discussion board and the internet for similar problems, but couldn't find something. Can anyone please help?

Here is the PR: https://github.com/DorisW3/capstone-project/pull/18

Thank you :-)

**Comment by Kheirah** [comment, marked as answer]
In `Navbar.js` you drop the `isActive` prop since you're not using it. Instead you define what the children of the `StyledLink` should be by checking for the current pathname. If it matches the "/favorites" string you show the filled star and the unfilled otherwise. You had a typo here as well; it's "favorites" with the "s" in the end.
```js
<StyledLink href={"/favorites"}>
  {router.pathname === "/favorites" ? <FilledStar /> : <Star />}
</StyledLink>
```

**Reply by DorisW3** [reply]
Thank you, it works!
![](https://emoji.slack-edge.com/TTHG21AH3/gratitude-thank-you/9dc842248f8e577f.png)
